cvmfs/mountpoint.cc

```diff
@@ -1761,13 +1761,29 @@ bool MountPoint::SetupBehavior() {
   LogCvmfs(kLogCvmfs, kLogDebug, "kernel caches expire after %d seconds",
            static_cast<int>(kcache_timeout_sec_));
 
-  bool hide_magic_xattrs = false;
+  MagicXattrManager::EVisibility xattr_visibility =
+    MagicXattrManager::kVisibilityAlways;
```

Comment:
Are you sure you don't want the default to be kVisibilityRootOnly?

Reply:
The default is `kVisibilityRootOnly` only if the `CVMFS_MAGIC_XATTRS_VISIBILITY` option is present.  Otherwise cvmfs sticks to the old default behavior.  However, this can still be changed of course in any of the config files.

Reply:
Right, but it seems to me that kVisibilityRootOnly is almost all the time going to be the preferred behavior, so there is an argument for making it be default if neither environment variable is set.

Reply:
Fair enough, see #2842 that changes the default in the devel branch (i.e., for the 2.10 release). For the patch release, I feel uncomfortable changing the client's default behavior.